spotify_to_ytmusic/settings.py

```diff
@@ -41,3 +41,4 @@ def __setitem__(self, section, key, value):
     def save(self):
         with open(self.filepath, "w") as f:
             self.config.write(f)
+        self.config.read(self.filepath)
```

Comment:
Not sure why this line is added, other than that it looks good to me

Reply:
Should be gone now, left it out by mistake.